Add Primitive data types article by Scaler

ReddiRani8496 opened the pull request:
Add article of primitive data types by Scaler.

ReddiRani8496 commented:
Please review it. Thank you

Shaluag123 commented on README.md:
```diff
@@ -163,6 +163,7 @@ Primitive values are directly assigned to a variable, and when you manipulate a
 -  [The differences between Object.freeze() vs Const in JavaScript — Bolaji Ayodeji](https://medium.com/@bolajiayodeji/the-differences-between-object-freeze-vs-const-in-javascript-4eacea534d7c)
 -  [Object to primitive conversion — JavaScript.info](https://javascript.info/object-toprimitive)
 - [Methods of primitives - Javascript.info](https://javascript.info/primitives-methods)
+-  [What are the Primitive Data Types in JavaScript ? - Scaler](https://www.scaler.com/topics/what-are-the-primitive-data-types-in-javascript/)
```
Please correct the indentation.

ReddiRani8496 commented:
Hi @Shaluag123 Line 165 has different indentation compared to 163 and 164. Which indentation I have to follow. Thank you

ReddiRani8496 commented:
Previously I added accoring to 164, now I changed to indentation according to line 165. Can you please review and let me know, if there are any changes. Thank you.

Shaluag123 commented:
now it is fine no further modification needed.

ReddiRani8496 commented:
Hi @leonardomso , can you please accept my Pull request. Thank you
